src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
I take that back. Looking at the list of warnings that GHC supports, the standard warnings seem completely reasonable. However, the additional warnings enabled by `-W` (-fwarn-unused-binds, -fwarn-unused-matches, -fwarn-unused-imports, -fwarn-incomplete-patterns, -fwarn-dodgy-exports, -fwarn-dodgy-imports) don't seem particularly useful with the exception of incomplete-patterns.

Incidentally, most of the new warnings are only triggered by `-W`, so maybe we can just remove that :)